src/Backend/Core/Layout/Sass/components/_pages.scss

```diff
@@ -179,15 +179,9 @@
     }
 
     .buttonHolder {
-      background: $white;
-      bottom: 0;
-      padding: 0 4px 4px 0;
-      position: absolute;
-      right: 0;
-      -webkit-transition: opacity 0.3s ease-in-out;
-      -moz-transition: opacity 0.3s ease-in-out;
-      -o-transition: opacity 0.3s ease-in-out;
-      transition: opacity 0.3s ease-in-out;
+      padding: 0.7em 0 0;
```

Comment:
Why not use px? we use pixels everywhere. This is not consistence.

Reply:
habit, @Katrienvh would you like me to change it?

Reply:
@carakas go ahead and change it